posthog/models/plugin.py

```diff
@@ -131,6 +131,7 @@ class PluginType(models.TextChoices):
     source: models.TextField = models.TextField(blank=True, null=True)
     latest_tag: models.CharField = models.CharField(max_length=800, null=True, blank=True)
     latest_tag_checked_at: models.DateTimeField = models.DateTimeField(null=True, blank=True)
+    capabilities: models.JSONField = models.JSONField(default=None, null=True)
```

Comment:
Could the default be an empty array? This way we can avoid `null=True` and only ever check whether the field `.includes` a capability.

Reply:
I think this should be an array, not an object, as alluded to here: https://github.com/PostHog/plugin-server/pull/384#issuecomment-843009662

Basically, we already need to store "type of capability" with the capability, and in the future I'm sure there will be more metadata we want to add (`capabilitiesCapturedAt` or whatever), making an object as the default a saner choice than an array.

No need to worry about indexing speed when querying as we'll have very few plugins anyway.

Reply:
Or a default object, right. Would be nice to avoid null checks. Unless we only fill this in for newly installed/updated plugins after merging this feature (using null as the "capabilities never captured yet" state), though should probably just detect this and save ASAP then to avoid incomplete states.

Reply:
Right, thanks, missed that.